dbt/models/default/default.vw_pin_appeal.sql

```diff
@@ -37,6 +47,11 @@ WITH reasons AS (
                 THEN REGEXP_REPLACE(htpar.user101, '[^[:alnum:]]', '')
         END AS reason_code3
     FROM {{ source('iasworld', 'htpar') }} AS htpar
+    WHERE htpar.cur = 'Y'
+        AND htpar.deactivat IS NULL
+        AND htpar.caseno IS NOT NULL
+        AND htpar.heartyp IN ('A', 'C')
```

Comment:
This conditional means we'll only get strictly defined appeals, I'm not entirely positive if that's what we want, but it is what we suggest our view is.